action/protocol/staking/handler_candidate_selfstake.go

```diff
@@ -0,0 +1,95 @@
+package staking
+
+import (
+	"context"
+	"math"
+	"math/big"
+
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+	"github.com/pkg/errors"
+
+	"github.com/iotexproject/iotex-core/action"
+	"github.com/iotexproject/iotex-core/action/protocol"
+	"github.com/iotexproject/iotex-core/pkg/util/byteutil"
+)
+
+const (
+	handleCandidateActivate = "candidateActivate"
+
+	candidateNoSelfStakeBucketIndex = math.MaxUint64
```

Comment:
remove since it's only used in test, and why is it needed in test?

Reply:
It will be used when register candidate without stake